.github/workflows/ci.yml

```diff
@@ -384,7 +388,8 @@ jobs:
     timeout-minutes: 150
     strategy:
       matrix:
-        variant: [ "clusterless,multi-site" ]
+        variant: [clusterless, multi-site]
+        testsuite: [ClusterlessTestSuite, MultisiteTestSuite]
```

Comment:
This will result in 4 runs:

* clusterless + ClusterlessTestSuite
* clusterless + MultisiteTestSuite
* ..
* ...

I would probably just do something in the `run new base tests` step that converts `clusterless` and `multi-site` to the classes to run